source/mainnet/net/guides/export-key.rst

```diff
@@ -0,0 +1,43 @@
+.. include:: ../../variables.rst
+.. _export-key:
+
+====================
+Export a private key
+====================
+
+In certain situations, such as testing interaction with smart contracts, you may need to export your private key for an account in your wallet. The key format is specifically for :ref:`import<concordium-client-import-accounts-keys>` to ``concordium-client``.
```

Comment:
The format in the file export is for the concordium-client, which I guess includes some more stuff in the .json file. The copyable key is "just" the key in itself.

Reply:
Is this something you want me to add here, or are you just pointing out something? I can't tell.